### ISSUE TITLE
accepted-claims-set.cddl can be removed

### ISSUE THREAD

**Issue by nedmsmith** [ietf-rats-wg/draft-ietf-rats-corim#340]
The set of intrep-*.cddl files replaces the need for accepted-claims-set.cddl.

**Comment by nedmsmith** [comment]
OK to close?

**Comment by thomas-fossati** [comment]
> Fixed by PR [#365](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/365) OK to close?

Yes.  I have added the `Fix: ...` annotation in the PR, therefore, it'll be automatically closed when #365 is merged.
